chainer/link.py

```diff
@@ -194,6 +194,15 @@ def __init__(self):
         finally:
             self._within_init_scope = old_flag
 
+    def __call__(self, *args, **kwargs):
+        try:
+            forward = self.forward
+        except AttributeError:
+            raise TypeError(
+                    '{} object does not have neither a method which overrides
+                    '\'Link.__call__\' nor a \'forward\' method.'.format(self))
```

Comment:
Hmm, I don't think this sentence is grammatically correct.

"{} object has neither `Link.__call__` method overridden nor `forward` method defined"?

Reply:
oops, right. I'll fix it